acceptance-tests/build.xml

```diff
@@ -13,8 +13,8 @@
 <project name="acceptance-tests" default="test" basedir="." xmlns:ivy="antlib:org.apache.ivy.ant">
     <property name="src.dir" location="src/main/java" />
     <property name="testsrc.dir" location="src/test/java" />
-    <property name="source.level" value="1.5" />
-    <property name="target.level" value="1.5" />
+    <property name="source.level" value="1.8" />
```

Comment:
Properties like this no longer vary per module, so ideally they'd be defined only once (once each for ant and maven).